### DISCUSSION TITLE
Required Zoom?

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#7] [Q&A] [answered]
Hi, 
I'm still reading through everything, so I apologize if these are answered in a place I haven't gotten to yet. I have two main questions. I didn't see a syllabus linked in Canvas under the Syllabus tab. Is it posted elsewhere? 
I'm also wondering if there are required Zoom meetings, or if everything is asynchronous.

**Comment by metasean** [comment, marked as answer]
@Skye404 - Thank you!  I had completely overlooked the Canvas syllabus.  Is is now posted https://usu.instructure.com/courses/731881/assignments/syllabus  Please let me know if you have any questions!

There are no _required_ Zoom meetings.  Other than this week, I will be available via zoom for 2 different hours each week.  This week it's 4 different hours, because there are inevitably technical issues and early semester questions. (I try to do different days and times, so that folks with different schedules have a chance to actually attend.)

If you want to "meet" via Zoom and there isn't a scheduled Office Hour that you can attend, reach out to me and I'll work with you to find a time that works.

If you're having any issues, I strongly suggest you come to an office hour sooner rather than later.  It's easy to run into a problem and in trying to solve it you end up going down a rabbit hole of wrong directions.  The sooner those hiccups are identified and corrected the better!

Last but not least, one of the reasons that I'm emphasizing constructive contributions, is that students with more experience won't benefit technically from working through every last exercise (they still need to do some of them each week, so that they can benefit from the instructional styles), but those more advanced students have an opportunity to help more junior students.  (e.g. "see one, do one, teach one.", "where one teaches, two learn")  I.e. junior students can get "credit" for asking questions and asking for help debugging, while more senior students can get "credit" for answering questions and helping the more junior students debug their code.

**Reply by Skye404** [reply]
Thanks, this is helpful!

**Comment by hardingjo** [comment]
@Skye404 Thanks for asking about the syllabus- I was wondering where it was too!
